apps/api-extractor/src/api/Extractor.ts

```diff
@@ -247,19 +249,43 @@ export class Extractor {
               convertLineEndings: NewlineKind.CrLf
             });
           }
-
-          apiReportChanged = true;
-        } else {
+       } else {
           messageRouter.logVerbose(ConsoleMessageId.ApiReportUnchanged,
-            `The API signature is up to date: ${actualApiReportShortPath}`);
+            `The API report is up to date: ${actualApiReportShortPath}`);
         }
       } else {
-        // NOTE: This warning seems like a nuisance, but it has caught genuine mistakes.
-        // For example, when projects were moved into category folders, the relative path for
-        // the API report files ended up in the wrong place.
-        messageRouter.logError(ConsoleMessageId.ApiReportMissing, `The API report file has not been set up.`
-          + ` Do this by copying ${actualApiReportShortPath}`
-          + ` to ${expectedApiReportShortPath} and committing it.`);
+        // The target file does not exist, so we are setting up the API review file for the first time.
+        //
+        // NOTE: People sometimes make a mistake where they move a project and forget to update the "reportFolder"
+        // setting, which causes a new file to silently get written to the wrong place.  This can be confusing.
+        // Thus we treat the initial creation of the file specially.
+        apiReportChanged = true;
+
+        if (!localBuild) {
+          // For a production build, issue a warning that will break the CI build.
+          messageRouter.logWarning(ConsoleMessageId.ApiReportNotCopied,
```

Comment:
>logWarning [](start = 24, length = 10)

Should the message here be different? Something about never having had created the file in a local build?

Reply:
Good point.

Reply:
Fixed